recipes/merlin/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: merlin
+  version: "1.1.2"
+
+source:
+  fn: merlin-1.1.2.tar.gz
+  url: http://csg.sph.umich.edu/abecasis/merlin/download/merlin-1.1.2.tar.gz
+  md5: 01ce17395aff3fba8d49447aeb2128fa
+  sha1: d5227276eae862506c7ed4475255c9be182585df
+  sha256: aa3c087b1c717209389705bf2ace4c24884575014a561dff250c0be93fe1e411
+
+requirements:
+  build:
+    - gcc   # [not osx]
+    - llvm  # [osx]
+    - zlib
+
+  run:
+    - libgcc    # [not osx]
+    - zlib
+
+test:
+  commands:
+    # WARNING - merlin do not return exit code 0 when it displays help
+    - bash -e +o pipefail -c 'merlin |grep "MERLIN 1.1.2"'
+
+about:
+  home: http://csg.sph.umich.edu/abecasis/merlin
+  license: OpenSource
```

Comment:
Merlin redistributes 'Mersenne twister code' in the file Random.cpp and it provides only this license.
I can replace 'OpenSource' by 'Free' or 'Mersenne Twister license' ?
because Mersenne Twister code is freely usable (see http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/MT2002/elicense.html)